docs/source/developer/contributing.rst

```diff
@@ -415,12 +421,12 @@ It is built with `Sphinx <http://sphinx-doc.org/>`_ and
 `numpydoc <https://numpydoc.readthedocs.io/>`_, which are available using
 conda or pip; i.e. ``conda install sphinx numpydoc``.
 
-To build the documentation, you need the bootstrap theme::
+To build the documentation, you need the bootstrap and rtd themes::
 
-   $ pip install sphinx_bootstrap_theme
+   $ pip install sphinx_bootstrap_theme sphinx-rtd-theme
```

Comment:
Two comments here. First, I thought this was mis-spelling, since my personal makefile contains `sphinx_rtd_theme`. But then I checked on anaconda.org and both `sphinx_rtd_theme` AND `sphinx-rtd-theme` exist.

<img width="980" alt="Screen Shot 2021-09-06 at 13 13 07" src="https://user-images.githubusercontent.com/41563/132208839-d006d8cb-debc-4796-b7b5-28689882769e.png">

<img width="980" alt="Screen Shot 2021-09-06 at 13 12 43" src="https://user-images.githubusercontent.com/41563/132208789-df7d8dfc-3e44-466f-8d4e-7694cde332f9.png">

Judging from downloads, I would tend to argue that `sphinx_rtd_theme` is probably what we want?

Also, I am not sure we actually need `sphinx_bootstrap_theme`. I do not have it in my environment:

<img width="744" alt="Screen Shot 2021-09-06 at 13 19 34" src="https://user-images.githubusercontent.com/41563/132209601-0ba4b623-4682-4bb7-8dab-dce03aab3b1a.png">

And it builds fine. IIRC I *think* we switched from the bootstrap to the rtd theme some time ago but the docs were not updated.

Reply:
Interestingly, when I read this, I even though it was a mispelling on my part, but when I went to https://github.com/readthedocs/sphinx_rtd_theme it turns out their instructions state `pip install sphinx-rtd-theme`. Probably fine to go with just `sphinx_rtd_theme` and leave out `sphinx_bootstrap_theme` since it doesn't seem to have been touched since 2019.

Reply:
Ah, I though this was `conda` not `pip`. Are we asking people to `pip` install into a conda environment? Do we want to recommend `conda` instead, since the packages are available there?

Reply:
Yeah, probably change this to `conda install`.